### PR TITLE
fix(lm-logs): fix timed out message from concat plugin causing data loss

### PR DESCRIPTION
**Pull request by siddharthck**
Use labels to reuse the route for common processing on logs.

### CODE REVIEW

**Comment by shubhashri12** [comment]
Looks good to me. please wait for another approval before merging.

**Review by choudharypooja — APPROVED** [review]
LGTM

**Review by shubhashri12 — APPROVED** [review]

File: charts/lm-logs/Chart.yaml
Comment (vkumbhar94) [inline charts/lm-logs/Chart.yaml:4]: Shouldn't it be `0.5.0`?

Comment (siddharthck) [reply charts/lm-logs/Chart.yaml:4]: 0.5.0 remained unreleased and it was patch fixed for helm-charts-qa

**Review by vkumbhar94 — APPROVED** [review]

**Comment by shubhashri12** [comment]
Merging this PR on behalf of @siddharthck

**Pull request merged**
